File: sample-ruby/sample.rb
require('../Sifter.rb')
template = Sifter.new
template.set_var('foo', 'bar')
template.set_var('condition', 'true')
template.set_var('array', [{'loop'=>1}, {'loop'=>2}, {'loop'=>3}])
template.display('sample.tmpl')

File: sample-php/sample.php
<?php

require('../Sifter.php');
$template = new Sifter;
$template->set_var('foo', 'bar');
$template->set_var('condition', 'true');
$template->set_var('array', array(array('loop'=>1), array('loop'=>2), array('loop'=>3)));
$template->display('sample.tmpl');

?>
